modules/t/VCFCollection.t

```diff
@@ -488,4 +488,21 @@ is_deeply(
   'get_all_Alleles_by_VariationFeature - dbSNP uses ref_freq_index()'
 );
 
+# Test get_all_clinical_significance_states() for VCF files
+$coll = $vca->fetch_by_id('ClnSig');
+ok($coll && $coll->isa('Bio::EnsEMBL::Variation::VCFCollection'), "fetch_by_id");
+
+my $temp = $coll->filename_template();
+$temp =~ s/###t\-root###/$dir/;
+$coll->filename_template($temp);
+ok($coll->filename_template =~ /^$dir/, "update filename_template");
+
+$slice = $sa->fetch_by_region('toplevel', 1, 10, 20);
+my $dont_fetch_vf_overlaps=1;
+my $vfs = $coll->get_all_VariationFeatures_by_Slice($slice, $dont_fetch_vf_overlaps);
+
+ok($vfs->[0]->get_all_clinical_significance_states()->[0] eq 'likely benign', 'get_all_clinical_significance_states - obtain single clinical significance entry');
+ok(scalar (@{$vfs->[1]->get_all_clinical_significance_states()}) eq 1, 'get_all_clinical_significance_states - ignore upsupported clinical significance entry');
+ok(scalar (@{$vfs->[2]->get_all_clinical_significance_states()}) eq 2, 'get_all_clinical_significance_states - obtain multiple clinical significance entries');
```

Comment:
`$vfs->[2]` only has one clinical significance -> 'likely benign'

Reply:
Yes, I was using an old version of the VCF file in this test. I have commited a new version of the VCF file which should hopefully fix.

Reply:
You loaded a new vcf file but kept the old files.

Reply:
One of the tests is still going to fail: `ok(scalar (@{$vfs->[2]->get_all_clinical_significance_states()}) eq 2` only has one valid clinical significance.

Reply:
Apologies, I comitted the wrong file. I used the .vcf file and not the .vcf.gz